src/main/java/seedu/manager/command/RemoveCommand.java

```diff
@@ -0,0 +1,22 @@
+package seedu.manager.command;
+
+public class RemoveCommand extends Command {
+    public static final String COMMAND_WORD = "remove";
+    public static final String REMOVE_SUCCESS = "Event removed successfully";
+    public static final String REMOVE_FAILURE = "Event not found";
```

Comment:
I think these public constants, if not used outside the class, can be made private?